service/src/main/java/bio/terra/cbas/models/GithubMethodSource.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.cbas.models;
+
+import java.util.UUID;
+
+public record GithubMethodSource(
```

Comment:
Sort of a nit (but I think naming is important so we should discuss if you disagree): this feels like it should be called `GithubMethodDetails`. 

I saw your argument in another comment that using the word "Source" sets us up to support non-Github methods, but I think it'll be just as clear to name some future method details record like `DockerhubMethodDetails`, `SomeothersourceMethodDetails`